theme/components/sections/home/banner.html.twig

```diff
@@ -1,4 +1,4 @@
-{% set bannerSrc = image.src|asset_url %}
```

Comment:
I added a fallback to the platform to handle this case, it will determine if the link is an uploaded image and if it is don't prepend the the asset url